spec/cob_index/suppress_config_spec.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+require "traject/array_writer"
+
+RSpec.describe "Suppress configuration" do
+  let(:indexer) do
+    settings = {
+      "nokogiri.each_record_xpath" => "//oai:record",
+      "nokogiri.namespaces" => { "oai" => "http://www.openarchives.org/OAI/2.0/" },
+      "solr.url" => "http://example.com/solr",
+      "writer" => Traject::ArrayWriter.new,
+    }
+
+    indexer = CobIndex::NokogiriIndexer.new(settings)
+    indexer.load_config_file("lib/cob_index/suppress_config.rb")
+    indexer
+  end
+  let(:oai_xml) do
+    <<~EOT
+    <?xml version="1.0" encoding="UTF-8"?>
+    <OAI-PMH xmlns="http://www.openarchives.org/OAI/2.0/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.openarchives.org/OAI/2.0/ http://www.openarchives.org/OAI/2.0/OAI-PMH.xsd">
+      <responseDate>2020-03-03T04:16:09Z</responseDate>
+      <request verb="ListRecords" metadataPrefix="marc21" set="blacklight" from="2020-03-02T20:47:11Z">https://na02.alma.exlibrisgroup.com/view/oai/01TULI_INST/request</request>
+      <ListRecords>
+      #{record_text}
+      </ListRecords>
+    </OAI-PMH>
+    EOT
+  end
+  let(:records_file) { StringIO.new(oai_xml) }
+  let(:records) { Traject::NokogiriReader.new(records_file, []) }
+  let(:record) { records.first }
+
+  before do
+    stub_const("ENV", ENV.to_hash.merge("SOLR_URL" => "http://example.com/solr"))
+  end
+
+
+  context "OAI record should be suppressed" do
```

Comment:
context here is "record should be suppressed" but then the record is skipped so it won't be suppressed.

Reply:
Yeah.. may bad that's the wrong context... should... context for first case is `status = something other than deleted` then expect record to be skipped and...
Will update verbiage.